src/Core/RevEng.Core.80/PatchedSqlServerDatabaseModelFactory.cs

```diff
@@ -748,6 +743,7 @@ private void GetColumns(
 (
     SELECT [t].[name], [t].[object_id], [t].[schema_id]
     FROM [sys].[tables] t
+    LEFT JOIN [sys].[extended_properties] AS [ep] ON [ep].[major_id] = [t].[object_id] AND [ep].[minor_id] = 0 AND [ep].[class] = 1 AND [ep].[name] = 'microsoft_database_tools_support'
```

Comment:
Why was this added here?

Reply:
because of the table filter that added anywhere